### DISCUSSION TITLE
Duplicate menu commands

### DISCUSSION THREAD

**Discussion by username116** [sumatrapdfreader/sumatrapdf#3741] [General]
It seems to me that there are duplicate commands in the menu and palette. I wondered while doing these translations in the menu:

In the top-left menu:
- File > Show in folder
- File > Open Directory in Explorer   -  (the difference is that the file is not selected in the folder in this case)

In the command palette Ctrl + K:
- Advanced Options...
- Advanced Settings...

With the latest pre-release v3.5.15699.

**Comment by kjk** [comment]
Yes, but it doesn't hurt and some people might find it one easier to find than the other.

**Comment by username116** [comment]
Thank you for the answer.

I close the discussion.
